packages/thirdweb/src/analytics/track.ts

```diff
@@ -20,10 +20,38 @@ export function trackConnect(args: {
   });
 }
 
+/**
+ * @internal
+ */
+export function trackPayEvent(args: {
+  client: ThirdwebClient;
+  event: string;
+  walletAddress?: string;
+  fromToken?: string;
+  fromAmount?: string;
+  toToken?: string;
+  toAmount?: string;
+  chainId?: number;
+  dstChainId?: number;
+}) {
+  track(args.client, {
+    source: "pay",
+    action: args.event,
+    clientId: args.client.clientId,
+    chainId: args.chainId,
+    walletAddress: args.walletAddress,
+    tokenAddress: args.fromToken,
+    amountWei: args.fromAmount,
+    dstTokenAddress: args.toToken,
+    dstChainId: args.chainId,
```

Comment:
hmmm you prob want to transform these into an existing column we have, or you will have to add new columns to usagev1 table

Reply:
We have these columns already don't we?